Source/VSTeam.psm1

```diff
@@ -31,18 +31,32 @@ if ($null -ne $env:TEAM_PROJECT) {
 
 
    # if not account and pat is set, then do not try to set the default project
-   if ($null -eq $env:TEAM_PAT -and $null -eq $env:TEAM_ACCT) {
-      Write-Warning "No PAT or Account set. You must set the environment variables TEAM_PAT or TEAM_ACCT before loading the module to use the default project."
+   if (($null -eq $env:TEAM_PAT -and $null -eq $env:TEAM_TOKEN) -and $null -eq $env:TEAM_ACCT) {
```

Comment:
```suggestion
   if (($null -eq $env:TEAM_PAT -or $null -eq $env:TEAM_TOKEN) -and $null -eq $env:TEAM_ACCT) {
```
I think you meant "or" not "and" because both would never be set. Description warning is also saying "or".